runelite-client/src/main/java/net/runelite/client/config/Keybind.java

```diff
@@ -60,7 +60,7 @@
 	private final int keyCode;
 	private final int modifiers;
 
-	public Keybind(int keyCode, int modifiers)
+	protected Keybind(int keyCode, int modifiers, boolean ignoreModifiers)
```

Comment:
maybe this constructor should be just keyCode, ignoreModifiers? as passing modifiers when you want to ignore them do not makes much sense. Or even just keyCode, even though that can maybe be a bit confusing idk

Reply:
But shift is also KeyCode i believe, and so you dont need to do that

Reply:
internally (and therefore through this constructor too) modifier only keys are stored as `VK_UNDEFINED` with modifiers set